package/MDAnalysis/lib/util.py

```diff
@@ -295,7 +295,7 @@ def anyopen(datasource, mode='r', reset=True):
                     stream.reset()
                 except (AttributeError, IOError):
                     try:
-                        stream.seek(0L)
```

Comment:
With long ints, won't this break when Py2 needed a Long int for some reason?  Iirc, sometimes the conversion doesn't always happen in time before overflows.